source/index.html

```diff
@@ -142,20 +143,21 @@ <h2 class="reviews__title">Отзывы</h2>
       </section>
 
       <section class="contacts">
+        <h2 class="contacts__title">Контакты</h2>
         <div class="contacts__wrapper">
-          <h2 class="contacts__title">Контакты</h2>
-          <span class="contacts__email">e-mail:</span>
+          <span class="contacts__email contacts__subtitle">e-mail:</span>
           <span class="contacts__email">info@mimimishkashop.ru</span>
-          <span class="contacts__addres">адрес</span>
-          <p class="contacts__addres">г. Санкт-Петербург ул. Большая Конюшенная д. 19/8, офис 101</p>
-          <div class="contacts__map">
+          <span class="contacts__addres contacts__subtitle">адрес:</span>
+          <p class="contacts__addres">г. Санкт-Петербург ул.&nbspБольшая Конюшенная д.&nbsp19/8, офис 101</p>
```

Comment:
```suggestion
          <p class="contacts__addres">г. Санкт-Петербург ул.&nbsp;Большая Конюшенная д.&nbsp19/8, офис 101</p>
```
Символ неразрывного пробела записывается так: `&nbsp;`. Все специальные символы HTML предваряются знаком `&` и заканчиваются знаком `;`.